### DISCUSSION TITLE
Enlarge click area for loader data to entire row

### DISCUSSION THREAD

**Discussion by hollandThomas** [Code-Forge-Net/Remix-Dev-Tools#19] [Ideas]
It would be a nice quality-of-life improvement when instead of these click areas
![Bildschirmfoto 2023-07-18 um 11 28 55](https://github.com/Code-Forge-Net/Remix-Dev-Tools/assets/20690465/6a0dbab7-16b8-4be0-a1de-9481170685df)

those could be enlarged to the entire row
![Bildschirmfoto 2023-07-18 um 11 28 55](https://github.com/Code-Forge-Net/Remix-Dev-Tools/assets/20690465/1c759965-dceb-443c-9a88-0329e78aab56)

**Comment by AlemTuzlak** [comment]
@hollandThomas good news, this will work in the upcoming release

**Comment by AlemTuzlak** [comment]
implemented
